Results-10-10-19.json

```diff
@@ -0,0 +1,12580 @@
+{
```

Comment:
I think we don't need this file anymore as there is the json file on the dummydata folder

Reply:
We may still need to set up a fetch request depending on Jon